VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimFileUpload/SelectFile/UploadFile/UploadFile.tsx

```diff
@@ -78,38 +79,59 @@ function UploadFile({ navigation, route }: UploadFileProps) {
 
   useEffect(() => {
     if (filesUploadedSuccess) {
-      navigateTo('FileRequest', { claimID: claim?.id || '' })
+      navigateTo('ClaimDetailsScreen', { claimID: claimID, claimType: ClaimTypeConstants.ACTIVE })
     }
-  }, [filesUploadedSuccess, claim, navigateTo])
+  }, [filesUploadedSuccess, claimID, navigateTo])
```

Comment:
scope creep? Why is this being changed? and shouldn't claimType be in the dependency array?

Reply:
okay i get the claimID part.

Reply:
ClaimType doesn't need to be. Since they can only upload on an active non closed claim we can use the constant for active, which doesn't need to be included